src/commands/create_drive.ts

```diff
@@ -25,15 +24,15 @@ new CLICommand({
 		DryRunParameter
 	],
 	async action(options) {
-		const context = new CommonContext(options, cliWalletDao);
-		const wallet: Wallet = await context.getWallet();
+		const parameters = new ParametersHelper(options, cliWalletDao);
```

Comment:
Looks like we don't need to supply a walletDao here anymore now that you're using it as a default

Reply:
There are a couple similar cases below

Reply:
34aa156

Reply:
Took care of it in 34aa156